music/src/ddsp/README.md

```diff
@@ -0,0 +1,85 @@
+# DDSP
+
+## Preset Checkpoints
+
+There are [4 DDSP Preset models](../../checkpoints) you can use, you can also create and use your own.
+
+Each of these models need some settings to be passed. You can tweak these settings, but these are the defaults:
+
+```
+  // violin
```

Comment:
Okay I've added all of these settings as `settings.json` files in each corresponding folder. I like that you provide documentation here on how to build with your own settings in the constructor, but I think the documentation and code should default to getting the URL of the folder (like in musicvae: https://github.com/magenta/magenta-js/blob/master/music/demos/music_vae.ts#L32), and then reading $URL/model.json and $URL/settings.json from the folder (sort of like in musicvae: https://github.com/magenta/magenta-js/blob/master/music/src/music_vae/model.ts#L847). The model should only error out if it didn't receive a settings in either the constructor or find and parse a settings.json in folder.

Reply:
Nice! I've modified the class to do this now.